frontend/src/components/TagInput/TagInput.tsx

```diff
@@ -91,8 +91,7 @@ export const TagInput = forwardRef<TagInputProps, 'input'>(
       (event: SyntheticEvent, tag: string) => {
         if (event.isDefaultPrevented()) return
         if (preventDuplicates && value.includes(tag)) return
```

Comment:
actually, might need to do some error handling here so tags are not duplicated when they are split

Reply:
> so tags are not duplicated when they are split

there is a preventDuplicates prop.

if preventDuplicates is true, then which behaviour do you mean?

1) if the new tags contain duplicates of the existing tags -> don't add any tag at all? (this follows existing behaviour)
or
2) if the new tags contain duplicates of the existing tags -> remove duplicate tags from new tags and add?

Reply:
option 2 looks better

Reply:
https://user-images.githubusercontent.com/102740235/191047376-47c969d9-c9da-49e8-8a4e-e1b19570ff64.mp4